<launch>
	<arg name="input"/>

	<group ns="odas">

		<node pkg="odas_ros" name="soundcard" type="src_soundcard" args="$(arg input)" output="screen">

			<param name="outTopic" value="raw_hops"/>

			<param name="config/fS" value="44100"/>
			<param name="config/hopSize" value="256"/>
			<param name="config/nChannels" value="8"/>
			<param name="config/nBits" value="16"/>
		</node>

	</group>
</launch>
